db/migrate/20180427130110_migrate_status_to_company.rb

```diff
@@ -0,0 +1,19 @@
+class MigrateStatusToCompany < ActiveRecord::Migration[5.1]
+  def change
+    Person.find_each do |person|
+      case person.status_id
+      when 1
+        person.update!(company_id: 1)
```

Comment:
it's always a bad idea relying on any primary keys. 
where are these companies created ?
shouldn't we first create the desired companies ?
how can we identify them ?

Reply:
We create the company with the seeder. Should we create them via migration?